include/mcu.hpp

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file mcu.hpp
+ *
+ * @brief MCU related
+ */
+
+#ifndef MCU_HPP
+#define MCU_HPP
+
+#include <cstdint>
+
+/*****************************************
+ * Public Function Prototypes
+ *****************************************/
+
+extern "C" {
+/**
+ * @brief Initializes System Clock.
+ * @note  Defined by cube.
+ */
+void SystemClock_Config(void);
+}
+
+namespace hal {
+class mcu {
+public:
+    /**
+     * @brief Initializes MCU and some peripherals.
+     */
+    static void init();
+
+    /**
+     * @brief Put the MCU to sleep.
+     *
+     * @param ms  Sleep time in milliseconds
+     */
+    static void sleep(uint32_t ms);
+};
+};  // namespace hal
```

Comment:
```suggestion
};
}  // namespace hal
```

linter achou isso redundante